src/main/java/com/powsybl/openloadflow/network/impl/LfLegBranch.java

```diff
@@ -46,25 +49,73 @@ public static LfLegBranch create(LfBus bus1, LfBus bus0, ThreeWindingsTransforme
         Objects.requireNonNull(bus0);
         Objects.requireNonNull(twt);
         Objects.requireNonNull(leg);
+
+        PiModel piModel = null;
+        PhaseControl phaseControl = null;
+
         double nominalV1 = leg.getTerminal().getVoltageLevel().getNominalV();
         double nominalV2 = twt.getRatedU0();
         double zb = nominalV2 * nominalV2 / PerUnit.SB;
         PhaseTapChanger ptc = leg.getPhaseTapChanger();
         if (ptc != null
                 && ptc.isRegulating()
                 && ptc.getRegulationMode() != PhaseTapChanger.RegulationMode.FIXED_TAP) {
-            LOGGER.error("3 windings transformer '{}' has a regulating phase tap changer which is not yet supported", twt.getId());
+            PhaseTapChanger.RegulationMode regulationMode = ptc.getRegulationMode();
+            PhaseControl.ControlledSide controlledSide = null;
+            if (ptc.getRegulationTerminal() == twt.getLeg1().getTerminal()) {
```

Comment:
I think the code only allows to regulate locally. It means that a phase tap changer located in a dedicated leg can only regulated at its local terminal. I think that next week I will increase the regulation capability to the 2 other terminals of the transformer (but I don't know if it is easy of not ;-)).